schema_from_diagram.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE patients (
+id INT GENERATED ALWAYS AS IDENTITY,
+name VARCHAR(50),
+date_of_birth date,
+PRIMARY KEY(id)
+);
+
+CREATE TABLE medical_histories (
+id INT GENERATED ALWAYS AS IDENTITY,
+admitted_at TIMESTMAP,
+patient_id INT,
+status VARCHAR(50),
+PRIMARY KEY (id),
+CONSTRAINT fk_patient FOREIGN KEY (patient_id) REFERENCES patients(id)
+);
+
+CREATE TABLE treatments (
+id INT GENERATED ALWAYS AS IDENTITY,
+type VARCHAR(100),
+name VARCHAR(100),
+PRIMARY KEY (id)
+);
+
+CREATE TABLE medical_histories_treatments (
+medical_history_id INT,
+treatment_id INT,
+CONSTRAINT fk_medical_history FOREIGN KEY(medical_history_id) REFERENCES medical_histories(id),
+CONSTRAINT fk_treatment FOREIGN KEY(treatment_id) REFERENCES treaments(id)
```

Comment:
- [ ] Adding indexes for all foreign keys. 
See example below for `medical_histories_treatments`
Please do it for all other foreign keys.
```sql
CREATE INDEX ON medical_histories_treatments (medical_history_id);
CREATE INDEX ON medical_histories_treatments (treatment_id);
```